d/drupal-warmer/drupal-warmer_ubi_8.5.sh

```diff
@@ -2,20 +2,20 @@
 
 # ----------------------------------------------------------------------------
 # Package          : drupal-warmer 
-# Version          : 2.0.3
+# Version          : 2.0.2, 2.0.3
 # Source repo      : https://git.drupalcode.org/project/warmer
-# Tested on        : UBI 8.5
-# Language         : PHP
-# Travis-Check     : True
-# Script License   : Apache License, Version 2 or later
-# Maintainer       : Bhagat Singh <Bhagat.singh1@ibm.com>
+# Tested on	: UBI: 8.5
+# Language      : PHP
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vishaka Desai <Vishaka.Desai@ibm.com>
```

Comment:
@vishakadesai please append your name instead of removing the name of the original contributor